Formula/flash.rb

```diff
@@ -0,0 +1,19 @@
+class Flash < Formula
+  desc "Command-line script to flash SD card images of any kind"
+  homepage "https://github.com/hypriot/flash"
+  url "https://github.com/hypriot/flash/releases/download/2.6.0/flash"
+  sha256 "38a8a5793ae5fac138eeff781c932de98977215b7bfd20e5f3e0a7f46920c6d7"
+
+  bottle :unneeded
+
+  def install
+    bin.install "flash"
+  end
+
+  test do
+    # Unfortunately there's not much else to test without actually
+    # downloading an image and/or writing it to an SD card
```

Comment:
You can download an image using a resource block.

Reply:
I can try to add a test like that, but the script then waits for the user to insert an SD card.  That is, unless I could mock `diskutil` somehow [like in flash's own Bats tests](https://github.com/hypriot/flash/blob/2.6.0/test/test_helper.bash#L19)?

Reply:
Ok, d4eb3b4aa36b8e93c92568a0ade97057767ad07b has a "real" test.  It mocks a few of the used binaries, but now it actually does something.  I also tried to add some helpful comments, but I'm not sure it's too specific regarding the actual implementation…